sudachipy/dictionarylib/dictionarybuilder.py

```diff
@@ -329,7 +329,7 @@ def parse_splitinfo(self, info):
             else:
                 ids.append(self.word_to_id(word))
                 if ids[-1] < 0:
-                    return ValueError('not found such a word')
+                    raise ValueError('not found such a word: {word}')
```

Comment:
@po3rin 
Thanks for fixing it!
I think you're going to use an `f-string`, but use it with `.format()` instead because SudachiPy supports python version 3.5 or higher.
The modified code will look like this.
```python
raise ValueError('not found such a word: {}'.format(word))
```
Could you fix this?

Reply:
Oops I'm sorry. I'll fix it right away.

Reply:
@t-yamamura 
done!